ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/NetHelper.java

```diff
@@ -101,6 +101,7 @@ public static String toNetType(final String contractType) {
         }
     }
 
+    //TODO delete once removed from all templates
```

Comment:
Its still valid.  I wrote code to replace its functionality, but I did not do a complete swap because the pull request was big enough as is.  I am going to do this as a separate refactor.